Stratis.Bitcoin/Features/Consensus/ConsensusFeature.cs

```diff
@@ -10,13 +10,14 @@
 using Stratis.Bitcoin.Connection;
 using Stratis.Bitcoin.Features.Consensus.CoinViews;
 using Stratis.Bitcoin.Features.Consensus.Deployments;
+using Stratis.Bitcoin.Interfaces;
 using Stratis.Bitcoin.Utilities;
 using System;
 using System.Threading;
 
 namespace Stratis.Bitcoin.Features.Consensus
 {
-    public class ConsensusFeature : FullNodeFeature, IConsensusFeature
+    public class ConsensusFeature : FullNodeFeature, IBlockDownloadState, INetworkDifficulty
```

Comment:
Same as other comment, perhaps we need to create a ConsensusManager that has this two interfaces 'IBlockDownloadState, INetworkDifficulty'